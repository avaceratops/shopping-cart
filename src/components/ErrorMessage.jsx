export default function Error() {
  return (
    <article className="grid h-72 place-content-center justify-items-center">
      <h1 className="mb-4 text-6xl font-bold">404</h1>
      <p className="italic text-gray-600">Page Not Found</p>
    </article>
  );
}
