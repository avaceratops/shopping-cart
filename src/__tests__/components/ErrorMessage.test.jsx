import { render } from '@testing-library/react';
import ErrorMessage from '../../components/ErrorMessage';

it('renders 404 message correctly', () => {
  const { container } = render(<ErrorMessage />);
  expect(container).toMatchSnapshot();
});
