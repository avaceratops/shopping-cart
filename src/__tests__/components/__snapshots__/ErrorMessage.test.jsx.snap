// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders 404 message correctly 1`] = `
<div>
  <article
    class="grid h-72 place-content-center justify-items-center"
  >
    <h1
      class="mb-4 text-6xl font-bold"
    >
      404
    </h1>
    <p
      class="italic text-gray-600"
    >
      Page Not Found
    </p>
  </article>
</div>
`;
